src/plugins/expression/expression-tile.tsx

```diff
@@ -33,6 +33,11 @@ export const ExpressionToolComponent: React.FC<ITileProps> = observer((props) =>
     });
   }
 
+  useEffect(() => {
+    if (mf.current?.readOnly === undefined) return;
```

Comment:
This seems error prone. 

I think from your comment you tried to pass a "readonly" attribute to the `<math-live` element. Is that right?
That is documented at the end here: https://cortexjs.io/mathlive/guides/static/
You might have just mis typed it in your comment but based on that doc the attribute has a dash in it: `read-only` (kebab case)

JSX won't let you pass a param with a dash like `read-only`. So you might have to pass it with `<math-live {...{"read-only": true}}`. 

Or React might automatically convert camel case to kebab case if you pass `<math-live readOnly`. React does this conversion with built in HTML elements. I'm not sure it will do it automatically with elements it doesn't know about.

Reply:
I found that I could use `read-only`, `readOnly`, and `readonly`, but whether I passed false or true booleans, both ended up making mathlive’s readOnly truthy.

The only way I could find to "mention" this property without it ending up `true` was to explicitly pass it `undefined`, so I did this:

```jsx
readOnly={props.readOnly === true ? true : undefined}
```

Let me know what you think - also mentioned in the slack.